package/ios/Core/CameraSession+CodeScanner.swift

```diff
@@ -74,6 +88,26 @@ extension CameraSession: AVCaptureMetadataOutputObjectsDelegate {
           "width": frame.size.width,
           "height": frame.size.height,
         ],
+        "corners": jsCorners
+      ]
+    }
+  }
+    
+  struct CodeScannerFrame {
+      
+    /**
+    Width of the scanned frame
+    */
+    let width: Int32
+    /**
+    Height of the scanned frame
+    */
```

Comment:
```suggestion
    let width: Int32
```

we don't really need comments for that tbh